sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,7 +1,18 @@
 ## Release History
 
-## 4.18.0-beta.1 (Unreleased)
+### 4.18.0-beta.1 (2021-08-22)
```

Comment:
The date is incorrect. It should be 
`2021-08-11`

Reply:
oops lol, updated